### PR TITLE
Add auto-labelling for automatic triage of PRs

### PR DESCRIPTION
**Pull request by palexster**
This PR adds a new pipeline to auto-add labels in new PRs. When a file is modified in a specific path, a corresponding label is added to contextualize the new contribution.

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+sig-api:
+  - operators/**/*
+
+sig-ui:
+  - webservice/**/*
+
+sig-operations:
+  - infrastructure/**/*
+  - provisioning/**/*
```
Comment (giorio94) [inline .github/labeler.yml:9]: Should we also add the user-environment label to the VM provisioning part?

Comment (palexster) [reply .github/labeler.yml:9]: Right!

**Pull request merged**